PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -193,9 +192,9 @@ protected function channel_close($args)
      * Confirm a channel close
      * Alias of AMQPChannel::do_close()
      *
-     * @param AMQPReader $args
+     * @param AMQPReader $reader
      */
-    protected function channel_close_ok($args)
+    protected function channel_close_ok(AMQPReader $reader)
```

Comment:
If you're worried about it, I can remove the type-hint and just leave the PHPDoc block